video_xblock/static/html/student_view.html

```diff
@@ -0,0 +1,13 @@
+{# To avoid interferrence with other video players and JavaScript modules (e.g. RequireJS) we wrap video player's code inside an iframe. #}
```

Comment:
e.g. RequireJS in Studio.
We can add: this can be removed, after Studio will drop RequireJS support. 
(It if is true and I understand correctly)